cli.js

```diff
@@ -24,10 +24,10 @@ if(options.args) {
   console.warn('Mocha-vite-puppeteer recieved args that don\'t match the supported syntax. Please check the github for syntax help if this was a mistake')
 };
 
-const jsonOptions = options.config ? JSON.parse(fs.readFileSync(options.config, 'utf-8')) : undefined;
-if(jsonOptions && jsonOptions.mvp) {
-  Object.keys(jsonOptions.map).forEach(key => {
-    options[key] = jsonOptions.mvp[key];
+const jsonOptions = options.config && options.config !== 'undefined' ? JSON.parse(fs.readFileSync(options.config, 'utf-8')) : undefined;
+if(jsonOptions) {
+  Object.keys(jsonOptions).forEach(key => {
+    options[key] = jsonOptions[key];
```

Comment:
I didn't notice this previously, but I think the normal way is for CLI options to take precedence over config file options, where this code makes it the other way around. If you agree, would you mind changing it?

Reply:
Ok, I've switched the order

I've removed the default key from the stdio getopts method, since that would specify a value and we'd use that over anything specified in the json. 

If we'd prefer to keep a default specified and then do some logic to prefer the json whenever the cli value matches the default value, that might be nicer. 
If we take that approach though, if a user explicitly specified a default value, it would be ignored in favor of a json value with the same key. E.g. if you set the port as 3006 in json and then try to manually set it to 3001 in the flags, 3001 would be ignored (since 3001 is the default).

Reply:
I like this implementation. Now the precedence is: CLI options OVER config file options OVER default values. Thx!